### ISSUE TITLE
Latex special character codes

### ISSUE THREAD

**Issue by oli-ver** [streetsidesoftware/vscode-spell-checker#556]
Is there a way to deal with LaTeX escaped special characters (i. e. umlauts)?
The spell checker should detect such characters and interpret them as the UTF-8 character, for example:

```
k\"{o}nnen
können
```
Right now these words cannot be checked:

![grafik](https://user-images.githubusercontent.com/965863/94357906-1bba6d00-009d-11eb-8c0d-fb8a3dd040e1.png)

Im my opinion it would suffice to offer a string search and replace feature for something like that where I can input patterns I want to replace before spell-check, for example:

```json
  "cSpell.search-replace": {
        "\"{o}": "ö",
        "\"{a}": "ä"
    }
```

Is there perhaps already something available that can be used?
